zammad/README.md

```diff
@@ -104,6 +102,39 @@ Open your browser on <http://localhost:8080>
 
 ## Upgrading
 
+### From chart version 10.x to 10.3.0
```

Comment:
Normaly the upgrade infos are only used for breaking changes.
I would put these infos along the config in the values.yaml.

Reply:
I will check this.

Reply:
I moved this to a new section below `Configuration` and changed it slightly.